### PR TITLE
EFA-460 Use single line for ellipsis to properly work

### PR DESCRIPTION
**Pull request by ig-garcia**

### CODE REVIEW

File: k9mail/src/main/res/layout/message_view_attachment.xml [hunk 1]
```diff
@@ -37,7 +37,7 @@
         android:layout_width="0dp"
         android:layout_height="wrap_content"
         android:ellipsize="middle"
-        android:maxLines="2"
+        android:singleLine="true"
```
Comment (Stereodark) [inline k9mail/src/main/res/layout/message_view_attachment.xml:40]: isn't this attribute deprecated?

Comment (ig-garcia) [reply k9mail/src/main/res/layout/message_view_attachment.xml]: Yes it is deprecated, I just updated it. Thanks for spotting that, I was just assuming that the ellipsis would not work properly with maxLines attribute but it seems to work alright when it is a single line.

Comment (ig-garcia) [reply k9mail/src/main/res/layout/message_view_attachment.xml]: Lots of issues with these attributes 🙃

Comment (Stereodark) [reply k9mail/src/main/res/layout/message_view_attachment.xml]: Google keeps changing their mind on APIs 🗡️

**Review by Stereodark — APPROVED** [review]

**Review by pzplanck — APPROVED** [review]

**Pull request merged**